examples/step-8/step-8.cc

```diff
@@ -598,13 +559,12 @@ namespace Step8
   // an initial grid that is totally unsuitable for the problem at hand.
   //
   // The unfortunate thing is that if the discrete solution is constant, then
-  // the error indicators computed by the <code>KellyErrorEstimator</code>
-  // class are zero for each cell as well, and the call to
-  // <code>refine_and_coarsen_fixed_number</code> on the
-  // <code>triangulation</code> object will not flag any cells for refinement
-  // (why should it if the indicated error is zero for each cell?). The grid
-  // in the next iteration will therefore consist of four cells only as well,
-  // and the same problem occurs again.
+  // the error indicators computed by the KellyErrorEstimator class are zero
+  // for each cell as well, and the call to
+  // Triangulation::refine_and_coarsen_fixed_number() on the will not flag any
+  // cells for refinement (why should it if the indicated error is zero for
+  // each cell?). The grid in the next iteration will therefore consist of
+  // four cells only as well, and the same problem occurs again.
```

Comment:
Can you please fix the "twice" in the paragraph below as well?

Reply:
It's been a long week. Where is the issue here?

Reply:
oh, we refine four times.

Reply:
Also fixed!